ui/design/src/components/Icon/new-icons/feedback.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const Feedback = (props: React.SVGProps<SVGSVGElement>) => (
   <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" {...props}>
     <g fill="none" fillRule="evenodd">
-      <rect width="20" height="20" />
+      {/* <rect width="20" height="20" /> */}
```

Comment:
this doesn't break the icon look somewhere else? if not then just delete the <rect>, otherwise create a new icon

Reply:
no breaking changes in the icon. was only used in the profile menu initially.